scripts/githooks/markdown-pre-commit.sh

```diff
@@ -26,11 +26,13 @@ set -e
 
 # ==============================================================================
 
-image_digest=3e42db866de0fc813f74450f1065eab9066607fed34eb119d0db6f4e640e6b8d # v0.34.0
+image_version=v0.34.0@sha256:230b1e0e0fa1c7dd6261e025cacf6761ac5ba3557a6a919eec910d731817ff28
```

Comment:
Was thinking about storing versions (and images) in a version file so they are all together - but this is probably cleaner and easier to find. What do we think?